releaser-core/src/main/java/releaser/internal/project/ProjectVersion.java

```diff
@@ -54,7 +54,7 @@
 	private static final Pattern SNAPSHOT_PATTERN = Pattern
 			.compile("^.*[\\.|\\-](BUILD-)?SNAPSHOT.*$");
 
-	private static final String MILESTONE_REGEX = ".*[\\.|\\-]M[0-9]+";
+	private static final String MILESTONE_REGEX = "^.*[\\.|\\-]M[0-9]+.*$";
```

Comment:
Why do we need `.*` at the end?
I guess, it is how it should be looking at snapshot and rc I just don't know what comes after.

Reply:
I was just making it look like the other ones.